src/index.js

```diff
@@ -20,10 +20,65 @@ function createElement(tagName, props, ...children) {
   return element;
 }
 
-function render() {
+const remove = (array, index) => [...array].filter((_, idx) => idx !== index);
+
+const calculator = (stack) => {
+  let memo = '';
+  stack.forEach((e) => { memo += e; });
+
+  return new Function(`return (${memo})`)();
+};
+
+const getCurrentNum = (stack) => {
+  let memo = '';
+  [...stack].reverse().some((e, idx) => {
+    if (/[0-9]/g.test(e)) memo = e + memo;
```

Comment:
```suggestion
    if (/[0-9]/g.test(e)) {
      memo = e + memo;
    }
```

if문 본문 한 줄로 쓰는 건 anti pattern 입니다.

Reply:
@wholemann 아 넵, 이것도 없앴습니다 ㅠ

if 한줄은 안티군요. 때론 간결해 보여서 좋았는데.
안쓰도록 하겠습니다~